test/e2e/network/service.go

```diff
@@ -791,11 +791,48 @@ var _ = SIGDescribe("Services", func() {
 		jig.TestReachableUDP(nodeIP, udpNodePort, framework.KubeProxyLagTimeout)
 
 		By("hitting the TCP service's LoadBalancer")
-		jig.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerCreateTimeout) // this may actually recreate the LB
+		jig.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerCreateTimeout)
 
 		if loadBalancerSupportsUDP {
 			By("hitting the UDP service's LoadBalancer")
-			jig.TestReachableUDP(udpIngressIP, svcPort, loadBalancerCreateTimeout) // this may actually recreate the LB)
+			jig.TestReachableUDP(udpIngressIP, svcPort, loadBalancerCreateTimeout)
+		}
+
+		By("Scaling the pods to 0")
+		jig.Scale(ns1, 0)
+		jig.Scale(ns2, 0)
+
+		//FIXME: prove ICMP-reject
```

Comment:
For the internal case (#72561) we ended up just assuming that we'd get "connection refused" rather than "timed out".

You could just check the counters on the iptables rules themselves; run `iptables-save -c` before and after, and verify that the packet count on the expected `REJECT` rule actually went up, and assume that if the right rule got hit, then the right end-user behavior happened too. (Though in fact, in this case the test would still pass even if the network plugin ate the ICMP reply...)

You could also check the global ICMP counters in `netstat -s`, which would let you confirm that the node had received an ICMP reject for *some* reason. That would have some unknowable number of false positives, but shouldn't have false negatives.